memory/org.eclipse.cdt.debug.ui.memory.search/src/org/eclipse/cdt/debug/ui/memory/search/MemorySearchResultsPage.java

```diff
@@ -297,4 +312,4 @@ public void setActionBars(IActionBars actionBars) {
 	@Override
 	public void setFocus() {
 	}
-}
+}
```

Comment:
(minor) The newline was lost at the end of this file - I don't know for sure if this suggestion from GitHub will fix it.

```suggestion
}
```